tests/integration/test_global.py

```diff
@@ -593,3 +593,79 @@ def test_global_list(pixi: Path, tmp_path: Path, dummy_channel_1: str) -> None:
         env=env,
         stdout_contains=["dummy-b: 0.1.0", "dummy-a: 0.1.0", "dummy-a, dummy-aa"],
     )
+
+
+def test_global_uninstall(pixi: Path, tmp_path: Path, dummy_channel_1: str) -> None:
+    env = {"PIXI_HOME": str(tmp_path)}
+    manifests = tmp_path.joinpath("manifests")
+    manifests.mkdir()
+
+    # Verify empty list
+    verify_cli_command(
+        [pixi, "global", "list"],
+        ExitCode.SUCCESS,
+        env=env,
+        stdout_contains="No global environments found.",
+    )
+
+    # Install dummy-b from dummy-channel-1
+    verify_cli_command(
+        [
+            pixi,
+            "global",
+            "install",
+            "--channel",
+            dummy_channel_1,
+            "dummy-a",
+            "dummy-b",
+            "dummy-c",
+        ],
+        ExitCode.SUCCESS,
+        env=env,
+    )
+    dummy_a = tmp_path / "bin" / exec_extension("dummy-a")
```

Comment:
You could also check for `dummy-aa`, which is another executable of package `dummy-a`.